nf_core/pipeline-template/workflows/pipeline.nf

```diff
@@ -16,6 +16,7 @@ for (param in checkPathParamList) { if (param) { file(param, checkIfExists: true
 
 // Check mandatory parameters
 if (params.input) { ch_input = file(params.input) } else { exit 1, 'Input samplesheet not specified!' }
+if (!params.outdir) { exit 1, 'Parameter outdir not specified!' }
```

Comment:
Do we need this? The parameter validation via the schema should fail if we don't specify '--outdir' ?

Reply:
You are right, did not realize that it was not set as a required parameter in the json schema of the chipseq pipeline...
Undo the change